docs/include/conf/past.rst

```diff
@@ -9,8 +9,8 @@ North America
 - `Write the Docs North America, May 5-6 2014, Portland, Oregon`_
 - `Write the Docs, Apr 8-9 2013, Portland, Oregon`_
 
-.. _Write the Docs North America, May 22-24 2016, Portland, Oregon: http://www.writethedocs.org/conf/na/2016/
-.. _Write the Docs North America, May 17-19 2015, Portland, Oregon: http://www.writethedocs.org/conf/na/2015/
+.. _Write the Docs North America, May 22-24 2016, Portland, Oregon: /conf/na/2016/
```

Comment:
`na/2016/` would work better locally :) Same for the other links you changed in the `past.rst` file.